packages/repack/js/NativeScriptManager.ts

```diff
@@ -0,0 +1,12 @@
+import type {TurboModule} from 'react-native';
+import {TurboModuleRegistry} from 'react-native';
+
+export interface Spec extends TurboModule {
+    readonly getConstants: () => {};
+
+    loadScript(scriptId: string, config: Object): Promise<Object>;
+    prefetchScript(scriptId: string, config: Object): Promise<Object>;
```

Comment:
Please use a proper type instead of `Object`

Reply:
as described in this [TypeScript to Native Type Mapping](https://reactnative.dev/docs/next/new-architecture-appendix#iii-typescript-to-native-type-mapping) doc: `Object` generates the equiv of `ReadableMap` type on Kotlin and `NSDictionary` on objc. and the definitions of these types weren't explicit, but it did generate as needed.